src/models/index.ts

```diff
@@ -11,34 +12,100 @@ if (!process.env.STORAGE_SECRET) {
 
 const encrypted = EncryptedField(Sequelize, process.env.STORAGE_SECRET);
 
-InstallationModel.init(
-	{
-		jiraHost: DataTypes.STRING,
-		secrets: encrypted.vault("secrets"),
-		sharedSecret: encrypted.field("sharedSecret", {
-			type: DataTypes.STRING,
-			allowNull: false
-		}),
-		clientKey: {
-			type: DataTypes.STRING,
-			allowNull: false
-		},
-	},
-	{ sequelize }
-);
+InstallationModel.init({
+	id: {
+		type: DataTypes.INTEGER,
+		primaryKey: true,
+		allowNull: false,
+		autoIncrement: true
+	},
```

Comment:
Are we changing Installations table somehow or we just adding id here which was missing?

Reply:
id was always there (look in sequelize migration scripts), but it just wasn't "registered" on the model as being there so we couldn't use it in our code.  This adds it in there and also adds validation to it so sequelize will throw and error if you try to create a new Installation with an id already set since it should be set by the DB.